### DISCUSSION TITLE
Search symbol in project

### DISCUSSION THREAD

**Discussion by li-yi-dong** [theniceboy/nvim#49] [Q&A]
Hi theniceboy:

Is there any short cut or extension in your configuration to facilitate symbol lookup?
I know the buildin :vimgrep, which is slow and unfriedly to use.

**Comment by theniceboy** [comment]
You may use `:Tags`, but I usually rely on LSP.

**Comment by dllvhaobo** [comment]
Is there any way to resize coc-reference windows size or float the coc-reference windows?

**Comment by ghost** [comment]
If your neovim's version is 5.0 or higher, you can use `lightspeed.nvim` to fast move.

I very very recommend this plugin, it's really fast!

It doesn't depend on anything! I think it better than `:Tags` and `vimgrep`!

There is its [repository](ggandor/lightspeed.nvim).

**Reply by ghost** [reply]
OK, This plugin is only valid for the current file. If you want to search word in all project, you can use `fzf` and its vim client `fzf.vim`. To search word, you also need to install `ripgrep`.

After, use `:Rg` to search.
